src/backend/distributed/operations/shard_rebalancer.c

```diff
@@ -863,6 +863,16 @@ replicate_table_shards(PG_FUNCTION_ARGS)
 }
 
 
+/*
+ * master_drain_node is a wrapper function for old UDF name.
+ */
+Datum
+master_drain_node(PG_FUNCTION_ARGS)
+{
+	return citus_drain_node(fcinfo);
+}
+
+
```

Comment:
later in this file we have strings containing master_move_shard_placement and master_copy_shard_placement, could you update those as well?